packages/invoices/src/OutboundShipment/api/api.ts

```diff
@@ -276,12 +293,10 @@ export const getOutboundQueries = (sdk: Sdk, storeId: string) => ({
 
     return result;
   },
-  deleteLines: (invoiceId: string) => async (ids: string[]) => {
+  deleteLines: async (lines: { id: string; invoiceId: string }[]) => {
```

Comment:
I wanted to consolidate upsert/delete to use an array of line objects so changed the param on this